apps/framework-cli/src/framework/core/infrastructure.rs

```diff
@@ -14,20 +14,24 @@
 //!
 //! If components need to reference each other, they should do so by reference and not by value.
 
+use crate::proto::infrastructure_map::{
+    infrastructure_signature, InfrastructureSignature as ProtoInfrastructureSignature,
+};
 use serde::{Deserialize, Serialize};
 
 pub mod api_endpoint;
 pub mod consumption_webserver;
 pub mod function_process;
 pub mod olap_process;
 pub mod orchestration_worker;
+pub mod sql_resource;
 pub mod table;
 pub mod topic;
 pub mod topic_sync_process;
 pub mod view;
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Eq)]
+#[serde(tag = "kind")]
```

Comment:
this this impact backwards compatibility?

Reply:
I don't think it does - from what I can tell, this structure was not used in the storage.

Reply:
It would impact backward compatibility between moose <> moose-lib but now that we enforce the versions are the same, I believe that should be fine.